assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -23,44 +23,51 @@ const DATE_RANGE = [{
     name: '1 year',
     period: 'P365D'
 }, {
-    name: 'Custom',
+    name: 'Custom dates/days',
     period: 'custom'
 }];
 const CUSTOM_TIMEFRAME_RADIO_BUTTON_DETAILS = [
+    {
+        id: 'custom-input-calendar-days',
+        value: 'calendar',
+        text: 'Calendar dates'
+    },
     {
         id: 'custom-input-days-before-today',
         value: 'days',
         text: 'Days before today'
-    },
-    {
-        id: 'custom-input-calendar-days',
-        value: 'calendar',
-        text: 'Calendar days'
     }
 ];
 
+// The standard 'selectedDateRange' options -- the options that are not a custom period, such as P129D or P3D
+const STANDARD_INITIAL_VALUES = ['custom', 'P7D', 'P30D', 'P1Y'];
```

Comment:
We don't use P1Y for 1 year, it is P365D.

Reply:
Yup. Thanks for pointing that out!